### PR TITLE
Add node flag

### PR DESCRIPTION
**Pull request by hchoura**

### CODE REVIEW

File: src/files.js [hunk 1]
```diff
@@ -384,6 +384,8 @@ FOAM_FILES([
   { name: "foam/physics/Collider" },
   { name: "foam/physics/PhysicsEngine" },
   { name: "foam/blob/Blob", flags: [ "java" ] },
+  { name: "foam/blob/BlobStore", flags: [ "java" ] },
```
Comment (hchoura) [inline src/files.js]: @kgrgreer Did we need to add node?

Comment (kgrgreer) [reply src/files.js]: yes

File: demos/swift/classes.js [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 var classes = [
   'foam.blob.Blob',
+  'foam.blob.BlobStore',
```
Comment (kgrgreer) [inline demos/swift/classes.js:9]: Not needed if they have flags: [ "java" ]

File: tools/classes.js [hunk 3]
```diff
@@ -83,6 +83,8 @@ var skeletons = [
 
  var proxies = [
    'foam.blob.Blob',
+   'foam.blob.BlobStore',
```
Comment (kgrgreer) [inline tools/classes.js]: Not needed if they have flags: [ "java" ]

**Pull request merged**